test/features/organization_membership_test.rb

```diff
@@ -0,0 +1,39 @@
+require "test_helper"
+
+class OrganizationMembershipTest < Capybara::Rails::TestCase
+
+  def user
+    @user ||= users :normal
+    @user.password = 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.